src/main/java/org/bytedeco/javacpp/tools/Parser.java

```diff
@@ -2355,6 +2355,10 @@ boolean function(Context context, DeclarationList declList) throws ParserExcepti
             }
         }
 
+        type = postDeclarator(context, decl, dcl, dcl.type);
```

Comment:
Maybe passing `type` instead of `dcl.type` is better ?

Reply:
Probably, does it change anything?

Reply:
No, they are the same at this line